assignment4.js

```diff
@@ -0,0 +1,17 @@
+let files = ["./f1","./f2","./f3", "./f4", "./f5"], i = 0;
+fs = require('fs');
+function callBackExample(file){
+  fs.readFile(file[i], 'utf8' , function (err,data) {
+    if (err) {
+	  return console.log(err);
+	}
+	console.log(data);
+	console.log('==================');
+	i++;
+	if(i<5) {
```

Comment:
there should be a space before and after an operator in JS. eg `a + b` instead of `a+b`